### DISCUSSION TITLE
How to prevent 'create another'?

### DISCUSSION THREAD

**Discussion by cleargoal** [filamentphp/filament#12586] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

In the docs, there is a snippet 
```
CreateAction::make()
    ->createAnother(false)
```
on the page https://filamentphp.com/docs/3.x/actions/prebuilt-actions/create#disabling-create-another

This snippet concerns **Action**.
I need the same effect but for the Resource.

How to reach it?

Help me please.

**Comment by leandrocfe** [comment, marked as answer]
`protected static bool $canCreateAnother = false;`

In the CreatePage.php

**Reply by cleargoal** [reply]
Great!
**Thank you very much!**

How do you know about everything in the Filament?

**Reply by leandrocfe** [reply]
No problem. I don't know everything, I read the docs and the source code haha
It's a win-win: I get to help out and learn something new at the same time.

**Reply by cleargoal** [reply]
Yes, you are right - it's win-win

**Reply by madushancs** [reply]
Hi leandrocfe, Could you please help me how to disable CreateAnother button only while uploading...
